src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1721,14 +1720,28 @@ bool canProceed()
     return proceed;
 }
 
+static int isSudo()
+{
+    const char* SudoMessage = "ERROR: root privileges required.";
+    if ((getuid() == 0) || (geteuid() == 0))
+        return 0;
+    std::cout << SudoMessage << std::endl;
```

Comment:
But why you need an intermediate variable at the first place?
```suggestion
    std::cerr << "ERROR: root privileges required." << std::endl;
```

Reply:
changed